### PR TITLE
Update copy and utternaces

### LINKED ISSUE #106: Update the 'abortStatement' and 'clarifyPrompts'
When clarifying, it might be good to say 'you can ask me 'what can you do?''

### PR DESCRIPTION
**Pull request by mindmelting**
Closes #106

### CODE REVIEW

**Comment by mindmelting** [comment]
# [Codecov](https://codecov.io/gh/mindmelting/lex-oscarbot/pull/141?src=pr&el=h1) Report
> Merging [#141](https://codecov.io/gh/mindmelting/lex-oscarbot/pull/141?src=pr&el=desc) into [master](https://codecov.io/gh/mindmelting/lex-oscarbot/commit/c38568638d53886af26db5a8e981f4f4056258b8?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/mindmelting/lex-oscarbot/pull/141?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #141   +/-   ##
=======================================
  Coverage   86.27%   86.27%           
=======================================
  Files          55       55           
  Lines         714      714           
=======================================
  Hits          616      616           
  Misses         98       98
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/mindmelting/lex-oscarbot/pull/141?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/mindmelting/lex-oscarbot/pull/141?src=pr&el=footer). Last update [c385686...c8bf80e](https://codecov.io/gh/mindmelting/lex-oscarbot/pull/141?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by dwmkerr** [comment]
Looks good - but we are inconsistent with how we italicise project names. Should we:

1. Always italicise project names (looks goofy in the CLI)
2. Always use single quotes like:
  > You are about to open an issue with the name 'issue name', and the details 'some details'?

File: lex/bot/OscarBot.json [hunk 1]
```diff
@@ -123,7 +123,7 @@
         "maxAttempts": 5,
         "messages": [
             {
-                "content": "Sorry, can you please repeat that?",
+                "content": "Dang, I couldn't understand that. Ask 'What can I ask you?' for phrases I will understand.",
```
Comment (dwmkerr) [inline lex/bot/OscarBot.json]: This is the prompt when they fail to elicit a slot, so they will not be able to ask 'what can I ask you' in this case. It's more 'can you check for typos and type it again?'

File: oscarbot/i18n/en.json [hunk 2]
```diff
@@ -12,34 +12,34 @@
...
   "myOpenIssuesResponse": "Your top five updated issues are: ${issueText}",
   "whatProjectsAmIWorkingOnResponse": "You own ${myReposCount} repositories and have contributed to ${contributedReposCount} repositories.",
-  "countOpenPRsResponse": "${projectName} has ${openPullRequests} open pull requests.",
-  "countIssuesResponse": "Jay-Z has 99 problems but ${projectName} only has ${issues}!",
-  "describeLastCommitResponse": "${author} was the most recent person to comment ${fromNow} on ${repository} and they said ${message}",
-  "getStarsResponse": "If ${projectName} keeps stealing all the stars, the night sky will soon be empty! It currently has ${stars} stars",
-  "getForksResponse": "${projectName} is big into kitchenware - it's got ${forks} forks",
+  "countOpenPRsResponse": "'${projectName}' has ${openPullRequests} open pull requests.",
+  "countIssuesResponse": "Jay-Z has 99 problems but '${projectName}' only has ${issues}!",
+  "describeLastCommitResponse": "${author} was the most recent person to comment ${fromNow} on '${repository}' and they said ${message}.",
```
Comment (dwmkerr) [inline oscarbot/i18n/en.json]: quotes around author and message

File: oscarbot/i18n/en.json [hunk 3]
```diff
@@ -12,34 +12,34 @@
...
+  "getStarsResponse": "If '${projectName}' keeps stealing all the stars, the night sky will soon be empty! It currently has ${stars} stars.",
+  "getForksResponse": "'${projectName}' is big into kitchenware - it's got ${forks} forks.",
   "elicitSlotGitHubUser": "What's your GitHub username?",
   "openIssueRequestTitle": "Sure, I'll need a title for that, what do you want to call it?",
-  "openIssueRequestDescription": "Great, I need you add a description for ${issueTitle}",
-  "openIssueConfirm": "I just want to confirm that you want to open an issue titled *${issueTitle}* and with description _${issueContent}_?",
-  "openIssueResponse": "Thanks for confirming, the issue is now open and it's located at ${url}",
-  "topIssuesResponse": "OK, here we go! The top issues for ${projectName} are: ${issueText}",
+  "openIssueRequestDescription": "Great, I need you add a description for ${issueTitle}.",
```
Comment (dwmkerr) [inline oscarbot/i18n/en.json]: quotes on issue title

**Pull request merged**